devpi_ldap/main.py

```diff
@@ -175,7 +175,7 @@ def _search(self, conn, config, **kw):
             if any(attribute_name in x.get('attributes', {}) for x in conn.response):
                 def extract_search(s):
                     if 'attributes' in s:
-                        return s['attributes'][attribute_name]
+                        return [s['attributes'][attribute_name]]
```

Comment:
The ldap3 author has stated that it now reads the schema and will return lists or single values accordingly. So this *is* required but the patch is wrong (does not handle lists).